pages/staking.jsx

```diff
@@ -587,46 +587,39 @@ const Staking = () => {
                     You have {tokenBalance} ergopad tokens.
                   </Typography>
                 )}
-                <Grid
-                  container
-                  spacing={3}
-                  alignItems="stretch"
-                  justifyContent="center"
-                  sx={{ flexGrow: 1 }}
-                >
-                  <Grid item md={10} xs={9}>
+                <Grid container sx={{mb: 2}} justifyContent={'space-between'} alignItems={'flex-start'}>
```

Comment:
Minor: Code formatting
```suggestion
                <Grid
                    container
                    sx={{ mb: 2 }}
                    justifyContent="space-between"
                    alignItems="flex-start"
                >
```